examples/mpns.rb

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+
+$:.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
+
+require 'ruby-push-notifications'
+
+device_urls = [
+  'First device url here',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.